icd/api/pipeline_compiler.cpp

```diff
@@ -1047,24 +1047,89 @@ void PipelineCompiler::GetPipelineCreationInfoNext(
 // =====================================================================================================================
 VkResult PipelineCompiler::SetPipelineCreationFeedbackInfo(
     const VkPipelineCreationFeedbackCreateInfoEXT* pPipelineCreationFeadbackCreateInfo,
-    const PipelineCreationFeedback*                pPipelineFeedback)
+    const VkGraphicsPipelineCreateInfo*            pCreateInfo,
```

Comment:
Passing VkGraphicsPipelineCreateInfo is going to be problematic when we have support for ray tracing pipelines.  I think you should pass uint32 stageCount and const VkPipelineShaderStageCreateInfo* pStages.